From e7b59bf36b9386fadd88de9c171d96ee34743928 Mon Sep 17 00:00:00 2001
From: Long Dang <long.dang.aj@rvc.renesas.com>
Date: Mon, 21 Nov 2016 15:49:41 +0700
Subject: [PATCH] StreamBoardLan:change video to upstream file 24FPS

Reason: current h264-vga-24.mp4 don't have correct framerate information for streaming.
This file generated from omxh264enc.

Signed-off-by: Long Dang <long.dang.aj@rvc.renesas.com>
---
 mmpoc-scripts/Video-Streaming-From-Board-Lan.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mmpoc-scripts/Video-Streaming-From-Board-Lan.sh b/mmpoc-scripts/Video-Streaming-From-Board-Lan.sh
index 3f86a77..b267889 100755
--- a/mmpoc-scripts/Video-Streaming-From-Board-Lan.sh
+++ b/mmpoc-scripts/Video-Streaming-From-Board-Lan.sh
@@ -4,7 +4,7 @@ IP_STREAM=$(cat /home/root/launcher-taskbar/mmpoc-scripts/IP_STREAM.info)
 
 /home/root/launcher-taskbar/mmpoc-scripts/IPCamera-Demo/banner 100 50 "Streaming..." &
 
-/home/root/RZ_scripts/RZ_GST-network-demo_transmit-video.sh /home/root/videos/h264-vga-24.mp4 $IP_STREAM
+/home/root/RZ_scripts/RZ_GST-network-demo_transmit-video.sh /home/root/videos/sintel_trailer-720p.mp4 $IP_STREAM
 
 killall -9 -q banner
 echo 3 > /proc/sys/vm/drop_caches
-- 
1.9.1

